lib/mparticle/models/device_information.rb

```diff
@@ -330,7 +346,9 @@ def list_invalid_properties
     # @return true if the model is valid
     def valid?
       platform_validator = EnumAttributeValidator.new('String', ["Unknown", "iOS", "Android", "tvOS"])
+      att_validator = EnumAttributeValidator.new('String', ["authorized", "denied", "not_determined", "restricted"])
       return false unless platform_validator.valid?(@platform)
+      return false unless att_validator.valid?(@att_authorization_status)
```

Comment:
just checking - we should allow you to not specify at all (i guess it's null) - can you test that that still works?

Reply:
Yep heres the lines that allows them to be valid and null. https://github.com/mParticle/mparticle-ruby-sdk/pull/13/files#diff-5b335611dd4fa2b388fe4c312aadeadbb532fb9f4f9f58b837f13eec91c24a0dL92